src/platform/user/tests/authorization/components/RequiredLoginView.unit.spec.jsx

```diff
@@ -8,9 +8,11 @@ import RequiredLoginView from '../../../authorization/components/RequiredLoginVi
 describe('<RequiredLoginView>', () => {
   const redirectFunc = sinon.spy();
   let oldWindow;
+  let oldSessionStorage;
 
   const initialSetup = () => {
     oldWindow = global.window;
+    oldSessionStorage = global.sessionStorage;
```

Comment:
Was this throwing a warning previously?

Reply:
Yes, this causes the new tests to fail, when removed.

Reply:
Huh. How odd. :man_shrugging: